### ISSUE TITLE
光流法去除动态点的时候运行结果没有给的图片结果那么好

### ISSUE THREAD

**Issue by xzh20000525** [Ewenwan/ORB_SLAM2_SSD_Semantic#31]

**Comment by evertale888** [comment]
Just want to ask, are map viewer & current window not running smooth when plotting octomap? Is this normal from the author design or is it just me?

**Comment by ghost** [comment]
yeah..optical flow seems doing nothing....not detecting dynamic point whatsoever.. & cannot run geometry as well (seg fault)
